src/Discord.Net.Commands/Readers/EnumTypeReader.cs

```diff
@@ -59,7 +61,7 @@ public override Task<TypeReaderResult> Read(CommandContext context, string input
                 if (_enumsByName.TryGetValue(input.ToLower(), out enumValue))
                     return Task.FromResult(TypeReaderResult.FromSuccess(enumValue));
                 else
-                    return Task.FromResult(TypeReaderResult.FromError(CommandError.ParseFailed, $"Value is not a {_enumType.Name}"));
+                    return Task.FromResult(TypeReaderResult.FromError(CommandError.ParseFailed, $"Name is not a {_enumType.Name}"));
```

Comment:
I'd leave this with the old error. It's talking about the value provided by the user.